API/hermes_node_api/hermes_node_api.cpp

```diff
@@ -507,7 +509,7 @@ class NodeApiStableAddressStack final {
  private:
   // The size of 64 entries per chunk is arbitrary at this point.
   // It can be adjusted depending on perf data.
-  static const size_t ChunkSize = 64;
+  static inline const size_t ChunkSize = 64;
```

Comment:
Without this change, I get an undefined symbol after adding libhermes as link target for hermesNodeApi

Reply:
My bad. I should have used the `constexpr` here. The `static inline` should work too, but `constexpr` may give it a better semantic.

Reply:
Pushed https://github.com/vmoroz/hermes-windows/pull/1/commits/affb6e4bf6ef59485aa8c2a58c4e83138b679cf9 👍 to use `static constexpr`.